.expeditor/config.yml

```diff
@@ -112,6 +112,13 @@ subscriptions:
       - trigger_pipeline:finish_release:
           post_commit: true
 
+  # Post release notification workloads
+  #
+  # pull_request_<ACTION>:<REPO>:<BRANCH>:<NUMBER>:<GITHUB_DELIVERY_UUID>
+  - workload: pull_request_opened:{{github_repo}}:expeditor/bump_rustfmt:*
+    actions:
+      - bash:.expeditor/scripts/finish_release/notify_rustfmt.sh
```

Comment:
This file appears to be missing; did you inadvertently leave it behind on your workstation?

Reply:
🤦‍♂ https://github.com/habitat-sh/habitat/pull/7258/commits/a4124785412f3501770b466b33ca8112bc825b32